README.md

```diff
@@ -30,26 +31,143 @@ From graph to a tree:
  - `npm i trampss-mst-onaction`
 
 ## API
-TODO
+### First try
+  1. Import the middleware from **trampss-mst-onaction** : `import onAction from 'trampss-mst-onaction'`
+  1. Write your reaction, the easiest way is to write it as a function :
```

Comment:
be careful, your ordered list has only `1`

Reply:
I did it on purpose because markdown engine will increment number on is own.
But it bugs with the code snippet so I'll fix it